cypress/support/profile.ts

```diff
@@ -141,3 +141,28 @@ export const addProfileTags = (tags: string[]) => {
   // wait for the tags to be added
   cy.get(`#tag-${tags[0]}`).should('be.visible');
 };
+
+export const unfollowUserByUsername = (username: string) => {
+  cy.get('#header-profile-pic').click();
+  cy.location('pathname').should('eq', '/profile');
+  cy.get('#profile-tab-following').click();
+  cy.contains('#list-profile-name', username)
+    .parentsUntil('#profile-list-root')
+    .parent()
+    .find('#list-unfollow-button')
+    .should('be.visible')
+    .click();
+  cy.get('#list-follow-button').should('contain.text', 'Follow');
+};
+
+export const verifyNotificationCounter = (expectedCount = '1') => {
+  // Wait and reload if the counter doesn't appear
+  cy.waitReloadWhileElementDoesNotExist('#header-notification-counter', 10);
+  // Assert the counter text
+  cy.get('#header-notification-counter').should('have.text', expectedCount);
+};
```

Comment:
Perhaps this one should be elsewhere, as it is not related to profile really. Maybe in `common.ts`?

Reply:
we could have a `header.ts`